scripts/validate-other-market-data.ts

```diff
@@ -73,6 +77,7 @@ interface Exchange {
 interface Params {
   id: number;
   ticker: string;
+  marketType: 'PERPETUAL_MARKET_TYPE_ISOLATED' | 'PERPETUAL_MARKET_TYPE_CROSS';
```

Comment:
should we make this into another type? but this is also fine -- just wonder if abacus(?) can just pass in the enum directly

Reply:
the string is coming from the otherMarketData.json